src/tests/migration.integration.test.ts

```diff
@@ -0,0 +1,78 @@
+import { join } from 'path';
+import { Sequelize } from 'sequelize';
+import { DotenvParseOutput } from 'dotenv';
+import Worker = require('tortoise');
+
+import * as iconv from 'iconv-lite';
+iconv.encodingExists('foo');
```

Comment:
what's this doing?

Reply:
i thot i was doing integration tests not chunks.